src/NuGetPackageVerifier/PackageIssueFactory.cs

```diff
@@ -259,5 +259,20 @@ public static PackageVerifierIssue IdIsNotOwned(string id, string[] allowedOwner
         {
             return new PackageVerifierIssue("PACKAGE_OWNERSHIP", id, $"The id '{id}' is not owned by one of {string.Join(", ", allowedOwners)}.", PackageIssueLevel.Error);
         }
+
+        public static PackageVerifierIssue DependencyVersionHasUpperBound(string id, string dependencyId)
+        {
+            return new PackageVerifierIssue("PACKAGE_DEPENDENCY", id, $"The version range of package dependency '{dependencyId}' for package '{id}' has an upper bound.", PackageIssueLevel.Warning);
```

Comment:
Think the id should be `PACKAGE_DEPENDENCY_VERSION_UPPER_BOUND`. We need to be able to exclude it from the verifier and consequently uniquely identify it